expose-app-api/database.js

```diff
@@ -0,0 +1,7 @@
+// database.js
+import { Sequelize } from 'sequelize';
+
+export const sequelize = new Sequelize('Exposedatabase', 'exposeswat', 'iamtheSwat1*', {
```

Comment:
non blocking - for capstone, I think this is OK, but in any prod system, you will never commit passwords like this

Reply:
how can it be done prooerly

Reply:
passwords should be stored in some sort of vault storage, it's a whole rabbit hole that we should avoid for now